codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/EventStreamUnmarshallTestCases.kt

```diff
@@ -261,10 +263,21 @@ internal object EventStreamUnmarshallTestCases {
                 "wrong-content-type",
                 br#"${testCase.validTestStruct}"#
             );
-            let result = ${format(generator)}().unmarshall(&message);
+            let result = $generator::new().unmarshall(&message);
             assert!(result.is_err(), "expected error, got: {:?}", result);
             assert!(format!("{}", result.err().unwrap()).contains("expected :content-type to be"));
             """,
         )
     }
 }
+
+fun conditionalBuilderInput(
```

Comment:
```suggestion
private fun conditionalBuilderInput(
```

Reply:
Made it `internal` in https://github.com/awslabs/smithy-rs/pull/2342/commits/e952961f0b3922c254961f114148f4173023a599. I couldn't make it private because it's used in the marshall tests as well.